lib/components/user/back-link.js

```diff
@@ -12,14 +12,18 @@ const StyledButton = styled(Button)`
   padding: 0;
 `
 
+export const backButtonContent = (
```

Comment:
Maybe this should be a component.

Reply:
Good call! Fixed in [127475c](https://github.com/opentripplanner/otp-react-redux/pull/1255/commits/127475c1cf79ef983aba3a5cf13f0cf6a255eb00)